lib/nerves_runtime/application.ex

```diff
@@ -20,6 +20,12 @@ defmodule Nerves.Runtime.Application do
 
     target = Nerves.Runtime.target()
 
+    _ =
+      if target != "host" do
+        # Load sysctl config file (/etc/sysctl.conf).
+        System.cmd("sysctl", ["-p"])
+      end
+
```

Comment:
If there's a failure, it would be nice to log the error. I also have a small preference to make the paths explicit even though I'm not too worried about this breaking. Perhaps a helper function like this:

```
  # call with load_sysctl_conf("/etc/sysctl.conf")

 defp load_sysctl_conf(path) do
    if File.exists?(path) do
      case System.cmd("/sbin/sysctl", ["-p", path]) do
        {_, 0} ->
          :ok
        {reason, _non_zero_exit} ->
          Logger.warn("Failed to run sysctl on #{path}: #{inspect(reason)}")
          false
      end
    else
      # Don't run sysctl if the conf file doesn't exist.
      :ok
    end

Reply:
👍 No problem! I've molded your recommendation into the style of the existing `try_entropy_generator` function so the codebase looks consistent, if that looks good to you.